Release/src/http/client/http_client_asio.cpp

```diff
@@ -482,14 +482,15 @@ class asio_context : public request_context, public std::enable_shared_from_this
 
             const auto &base_uri = m_context->m_http_client->base_uri();
             const auto &host = utility::conversions::to_utf8string(base_uri.host());
-            const auto &port = base_uri.port();
+            const int portRaw = base_uri.port();
+            const int port = (portRaw != 0) ? portRaw : 443;
 
             std::ostream request_stream(&m_request);
             request_stream.imbue(std::locale::classic());
 
-            request_stream << "CONNECT " << host << ":" << ((port != 0) ? port : 443) << " HTTP/1.1" << CRLF;
-            request_stream << "Host: " << host << ":" << ((port != 0) ? port : 443) << CRLF;
-            request_stream << "Proxy-Connection: Keep-Alive" << CRLF;
+            request_stream << "CONNECT " << host << ":" << port << " HTTP/1.1\r\n";
```

Comment:
May I ask what is the reason for using both CRLF and hardcoded "\r\n" in this code snippet?

Reply:
When CRLF was the entire constant, I left it alone; worst case it saves a strlen on the crlf. Where CRLF was causing an additional trip through <<, I removed it.